production/technical_userguide.md

```diff
@@ -67,15 +65,44 @@ Client Secret:
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-In the following cURL command, notice that we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which requires base-64 encoding be performed on your clientId and secret**
+
+Format:
 
 ```
-curl -d '' -X POST "https://api.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic MDk4NjlhN2YtNDZjZS00OTA4LWE5MTQtNjEyOWQwODBhMmFlOjY0OTE2ZmU5NmY3MWFkYzc5YzU3MzVlNDlmNGU3MmYxOGZmOTQxZDBkZDYyY2Y0M2VlMWFlMDg1N2UyMDRmMTczYmExMGU0MjUwYzEyYzQ4"
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
 ```
 
-**Response**
+In the following cURL command, we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
```

Comment:
Looks like there are some extraneous double-quotes in this sentence, around the colon.
Screenshot from local rendering:

<img width="660" alt="Screen Shot 2020-04-27 at 10 48 03 PM" src="https://user-images.githubusercontent.com/37818548/80441641-4aa79980-88d9-11ea-934a-80f4574815e7.png">

Reply:
Accepted, resolved.